ui/src/app/utils/resourceActions.ts

```diff
@@ -32,36 +33,36 @@ export function toDisplay(resourceType: ResourceType): string {
 
 export interface ConvertToResourcesArgs {
   list:  FileDetail[] | Cohort[] | CohortReview[] | ConceptSet[] | DataSet[];
-  workspaceNamespace: string;
-  workspaceId: string;
-  accessLevel: WorkspaceAccessLevel;
   resourceType: ResourceType;
+  workspace: WorkspaceData;
 }
 
 export function convertToResources(args: ConvertToResourcesArgs): WorkspaceResource[] {
   const resourceList = [];
   for (const resource of args.list) {
-    resourceList.push(convertToResource(resource, args.workspaceNamespace, args.workspaceId,
-      args.accessLevel, args.resourceType));
+    resourceList.push(convertToResource(resource, args.resourceType, args.workspace));
   }
   return resourceList;
 }
 
-export function convertToResource(resource: FileDetail | Cohort | CohortReview | ConceptSet
-  | DataSet, workspaceNamespace: string, workspaceId: string,
-  accessLevel: WorkspaceAccessLevel,
-  resourceType: ResourceType): WorkspaceResource {
+export function convertToResource(
+  resource: FileDetail | Cohort | CohortReview | ConceptSet | DataSet,
+  resourceType: ResourceType,
+  workspace: WorkspaceData): WorkspaceResource {
+  const {namespace, id, accessLevel, cdrVersionId, billingStatus} = workspace;
   let modifiedTime: string;
   if (!resource.lastModifiedTime) {
     modifiedTime = new Date().toDateString();
   } else {
     modifiedTime = new Date(resource.lastModifiedTime).toString();
   }
   const newResource: WorkspaceResource = {
-    workspaceNamespace: workspaceNamespace,
-    workspaceFirecloudName: workspaceId,
+    workspaceNamespace: namespace,
+    workspaceFirecloudName: id,
     permission: WorkspaceAccessLevel[accessLevel],
-    modifiedTime: modifiedTime
+    modifiedTime,
+    cdrVersionId,
```

Comment:
This line is the source of the problem - all the rest is stronger typing and signature changes.